build-tools/makefile_components/base_push.mak

```diff
@@ -6,7 +6,7 @@
 
 push: .push-$(DOTFILE_IMAGE) push-name
 .push-$(DOTFILE_IMAGE): .container-$(DOTFILE_IMAGE)
-	docker push $(DOCKER_REPO):$(VERSION)
```

Comment:
Same, no need to change, but I'm assuming this code doesn't get included any more.